samples/jvm/junit5/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore Gradle project-specific cache directory
+.gradle
+
+# Ignore Gradle build output directory
+build
```

Comment:
that's good but unfortunately you have already `build/...` files in your commit. Please remove them